app/code/Magento/Review/Model/Rating.php

```diff
@@ -97,7 +101,7 @@ public function addOptionVote($optionId, $entityPkValue)
      * @param int $optionId
      * @return $this
      */
-    public function updateOptionVote($optionId)
+    public function updateOptionVote(int $optionId): Rating
```

Comment:
Please remove param strict types to ensure backward compatibility (i.e. $optionId can be currently passed as a string by third-party extension)